schema/deploy/shipping/views.sql

```diff
@@ -593,3 +592,82 @@ create or replace view shipping.metadata_for_augur_build_v3 as
 
 comment on view shipping.metadata_for_augur_build_v3 is
 		'View of metadata necessary for SFS augur build';
+
+
+create or replace view shipping.sample_with_best_available_encounter_data_v1 as
+
+    with specimen_manifest_data as (
+        select distinct on (sample_id)
+            sample_id,
+            date_or_null(document->>'date') as collection_date,
+            trim(both ' ' from document->>'swab_site') as swab_site,
+            trim(both ' ' from document->>'sample_origin') as sample_origin
```

Comment:
I'm confused why these have to be pulled from `receiving.manifest`. They should all be within `sample.details`

Reply:
I can't remember why I originally used `receiving.manifest` for this view. I may not have realized these details were also in `sample.details`...

Reply:
OK, I rewrote the query to use `sample.details` over `receiving.manifest`.